Plots/Contours/NCL_conLev_3.py

```diff
@@ -0,0 +1,114 @@
+ds = xr.open_dataset(gdf.get("netcdf_files/Tstorm.cdf"))
+
+# Extract temperature data at the first timestep
+T = ds.t.isel(timestep=0, drop=True)
+
+###############################################################################
+# Plot:
+
+# Generate figure (set its size (width, height) in inches)
+plt.figure(figsize=(8, 8))
+ax = plt.axes()
+
+# Import an NCL colormap
+newcmp = 'plasma'
+
+# Contourf-plot data (for filled contours)
+num_lev = 16  # Number of levels
+temp = T.plot.contourf(ax=ax, vmin=244, vmax=308, levels=np.linspace(244, 308, num_lev + 1),
+                       cmap=newcmp, add_colorbar=False, add_labels=False)
+# Contour-plot data (for line contours)
+T.plot.contour(ax=ax, vmin=244, vmax=308, levels=np.linspace(244, 308, num_lev + 1),
+               colors='black', linewidths=0.5, add_labels=False)
+
+# Add horizontal colorbar
+cbar_ticks = np.arange(248, 308, 4)
+cbar = plt.colorbar(temp, orientation='vertical', pad=0.005)
+cbar.ax.tick_params(labelsize=11)
+cbar.set_ticks(cbar_ticks)
+
+# Use geocat.viz.util convenience function to set axes tick values
+gvutil.set_axes_limits_and_ticks(ax, xlim=(-140, -50), ylim=(20, 60),
+                                 xticks=[-135, -90],
+                                 yticks=np.arange(20, 70, 10))
+
+# Use geocat.viz.util convenience function to make plots look like NCL plots by using latitude, longitude tick labels
+gvutil.add_lat_lon_ticklabels(ax)
+
+# Remove the degree symbol from tick labels
+ax.yaxis.set_major_formatter(LatitudeFormatter(degree_symbol=''))
+ax.xaxis.set_major_formatter(LongitudeFormatter(degree_symbol=''))
+
+# Use geocat.viz.util convenience function to add minor and major tick lines
+gvutil.add_major_minor_ticks(ax, x_minor_per_major=3, y_minor_per_major=5,
+                             labelsize=12)
+
+# Remove ticks on right side
+ax.tick_params(which='both', right=False)
+
+# Use geocat.viz.util convenience function to add title
+gvutil.set_titles_and_labels(ax, maintitle="Explanation of Python contour levels")
+
+# Create labels by colorbar
+size = 8
+y = 1 / num_lev / 2  # Offset from x axis in axes coordinates
+ax.text(0.949, y, 'T < 248', fontsize=size, horizontalalignment='center',
+        verticalalignment='center', transform=ax.transAxes,
+        bbox=dict(boxstyle='square, pad=0.25',
+                  facecolor='papayawhip',
+                  edgecolor='papayawhip'))
+text = '{} <= T < {}'
+for i in range(0, 14):
+    y = y + 1 / num_lev  # Vertical spacing between the labels
+    ax.text(0.904, y, text.format(cbar_ticks[i], cbar_ticks[i + 1]),
+            fontsize=size, horizontalalignment='center',
+            verticalalignment='center', transform=ax.transAxes,
+            bbox=dict(boxstyle='square, pad=0.25',
+                      facecolor='papayawhip',
+                      edgecolor='papayawhip'))
+y = y + 1 / num_lev  # Increment height once more for top label
```

Comment:
```suggestion

y = y + 1 / num_lev  # Increment height once more for top label
```
Added white space for clarity